frontend/playground-frontend/src/components/Home/CSVInputFile.js

```diff
@@ -92,6 +132,9 @@ const CSVInputFile = (props) => {
 CSVInputFile.propTypes = {
   setData: PropTypes.func.isRequired,
   setColumns: PropTypes.func.isRequired,
+  setOldData: PropTypes.func.isRequired,
+  fileName: PropTypes.any,
```

Comment:
Please declare the type of fileName. Maybe `PropTypes.string`?

Reply:
@farisdurrani I tried doing that but I don't think PropTypes.string allows null values, causing a compile error, so I had to use PropTypes.any to resolve that

Reply:
That's odd. Not putting `.isRequired` should allow null values but I trust you